content/en/references/changelog/index.md

```diff
@@ -11,6 +11,7 @@ hide_readingtime: true
 
 | Version   | Changes                                                                                                              |
 |-----------|-----------------------------------------------------------------------------------------------------------------------|
+| 0.2.3     | Add initial support for `OBJECT_KEYS`/`PARSE_IP`/`DESCRIBE FUNCTION` functions; add support for `DATE_TRUNC`/`NVL2`/`LEAST`/`GREATEST`/`COALESCE`/`SPLIT`/`FLOOR`/`DATE_FROM_PARTS`/`LAST_QUERY_ID`/`INITCAP`/`LEAD`/`LAG`/`DATEDIFF`/`TIMEDIFF`/`HASH`/`ANY_VALUE`/`CONTAINS`/`BETWEEN`/`MODE`/`AVG`/`CBRT`/`CEIL`/`ZEROIFNULL`/`ARRAY_UNIQUE_AGG`/`DEGREES`/`RADIANS`/`EXP`/`REPEAT`/`REVERSE`/`SQRT`/`ASCII`/`LOG`/`LN` function; enhance parity for creation/deletion of schemas with fully qualified names; enhance parity for inserting timestamps with subsecond precision; enhance parity for CTAS with nested subqueries; enhance parity for id placeholders in JDBC prepared statements; enhance parity for metadata queries and schema lookup with fully qualified table names; `MIN_BY`/`MAX_BY` aggregate functions adjustments; Properly extract db/schema parameters for JDBC connections; Implement trigonometric and hyperbolic functions; Add support for GET stage files |
```

Comment:
Noticed `IS_NULL_VALUE` missing from the list of supported functions as part of the new release. If you can please include it as well.

Reply:
Great catch @eruditmorina!!! 🙌 

I missed that addition because I reviewed the commit history using `git log v0.2.2..v0.2.3 --oneline`, and the `IS_NULL_VALUE` was added in commit localstack/snowflake@80e9de2. The commit message, `handle json null values query processor` wasn’t very clear about the addition.

Just a small nitpick: the commit message could have been more descriptive in this case. We should keep in mind that release notes are often generated based on commit messages, which will be especially important once we introduce automated release notes https://www.notion.so/localstack/Snowflake-Task-Board-42b0c52fd81d4248bc4573e41b580d9c?p=113fc2a2343180ae9d5ce91f4af647ed&pm=s